docs/Reply.md

```diff
@@ -177,8 +177,12 @@ fastify.get('/', function (request, reply) {
   reply.send(httpErrors.Gone())
 })
 ```
+By default Fastify will log the error depending on the `statusCode`:
+- for `400 <= statusCode < 500` the error will be logged with level `info`
+- for `statusCode >= 500` the error will be logged with level `error`
 
-If you want to completely customize the error response, checkout [`setErrorHandler`](https://github.com/fastify/fastify/blob/master/docs/Server.md#seterrorhandler) API.
+If you want to completely customize the error handling, checkout [`setErrorHandler`](https://github.com/fastify/fastify/blob/master/docs/Server.md#seterrorhandler) API.<br>
+*Note: you are responsibile for logging when setting the error handler*
```

Comment:
We need to add some docs that show that in the error handler we log the error.

Reply:
I added a code example below and a comment in Server.md